virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -47,6 +47,8 @@ Without an external monitoring system or a qualified human monitoring node healt
 ** Intel VT or AMD-V hardware virtualization extensions are enabled
 ** The no-execute (NX) flag is enabled
 
+* If FIPS mode is xref:../../installing/installing-fips.adoc#installing-fips[set up correctly at the OpenShift level], it is automatically enabled and no additional setup is needed for {VirtProductName}. FIPS must be enabled before your cluster's operating system boots for the first time.
```

Comment:
**Suggested rewrite:** If FIPS mode is xref:../../installing/installing-fips.adoc#installing-fips[enabled for your cluster], no additional setup is needed for {VirtProductName}. Support for FIPS cryptography must be enabled before the operating system that your cluster uses boots for the first time.